.github/workflows/opendata-ci.yml

```diff
@@ -0,0 +1,42 @@
+name: Open Data CI
+on:
+  pull_request:
+    branches:
+      - dev
+    paths:
+      - 'src/open-data/**/*.csv'
+      - 'src/open-data/**/*.json'
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env: 
+      CI_COMMIT_MESSAGE: Update open data
+      CI_COMMIT_AUTHOR: Open Data CI
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        ref: ${{ github.event.pull_request.head.ref }}
+        token: ${{ secrets.WORKFLOW_GIT_ACCESS_TOKEN }}
+    - name: Set environment variable "commit-message"
+      run: echo "commit-message=$(git log -1 --pretty=format:'%s')" >> $GITHUB_ENV
```

Comment:
Is this setting the commit message to the same as the last one? Should we just use something static like "Run Build-OpenData -PowerShell" or "Generate PowerShell commands for open data"?

Reply:
it's currently set as "Update open data" in the env vars, I can update it to be a little more specific

Reply:
Updated to "Generate PowerShell commands for open data"